java/ql/lib/semmle/code/xml/XML.qll

```diff
@@ -174,7 +177,10 @@ class XmlDTD extends XmlLocatable, @xmldtd {
   }
 }
 
-/** DEPRECATED: Alias for XmlDTD */
+/** DEPRECATED: Alias for XmlDtd */
+deprecated class XmlDTD = XmlDtd;
+
+/** DEPRECATED: Alias for XmlDtd */
 deprecated class XMLDTD = XmlDTD;
```

Comment:
This was just  introduced in your prioir PR, so I think we can combine these aliases.
```suggestion
/** DEPRECATED: Alias for XmlDtd */
deprecated class XMLDTD = XmlDtd;
```

Reply:
Same for C++.

Reply:
> Same for C++.

And the other languages, I'm fixing all in one go.